include/wx/string.h

```diff
@@ -75,6 +75,7 @@
 
 class WXDLLIMPEXP_FWD_BASE wxString;
 
+
```

Comment:
AFAICT, all the changes in this file are accidental whitespace changes and as such they should be reverted,  so the file is not included in the eventual final commit.